### PR TITLE
Add tests for RevUserController

### PR DESCRIPTION
**Pull request by RyanBusby**
Disabled the tests that should pass but don't due to the `expect(jsonPath("$").isNotEmpty())` problem.
There are also some updates to the URIGuide in this pull request.

### CODE REVIEW

**Review by DDRIV — APPROVED** [review]
documentation present, tests look like tests

File: src/test/java/com/revature/RevAssure/controller/RevUserControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        mockMvc.perform(post("/revuser/register")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(new ObjectMapper().writeValueAsString(revUser)))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath("$").isNotEmpty())
+                .andReturn();
+    }
+
+    @Test
+    @Disabled
```
Comment (cptvictor) [inline src/test/java/com/revature/RevAssure/controller/RevUserControllerTest.java:93]: comment out/get rid of @Disabled, comment out troublesome lines instead

**Review by cptvictor — CHANGES_REQUESTED** [review]
don't use @Disabled, comment out bad lines instead

File: src/test/java/com/revature/RevAssure/controller/RevUserControllerTest.java (hunk 1 above)
Comment (RyanBusby) [reply src/test/java/com/revature/RevAssure/controller/RevUserControllerTest.java]: changes have been made.

**Review by cptvictor — APPROVED** [review]
lgtm

**Review by DDRIV — APPROVED** [review]
same as before

**Pull request merged**